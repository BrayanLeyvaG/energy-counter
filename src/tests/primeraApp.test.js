import React from 'react';
import '@testing-library/jest-dom'
import {shallow} from 'enzyme'
// import {render} from '@testing-library/react'
// import '@testing-library/jest-dom/extend-expect'

import PrimeraApp from "../PrimeraApp"

 
 
 describe('Pruebas en <Primera App />', () => {
/*      test('debe de mostrar el mensaje "Hola soy Brayan"', () => {
        
        const saludo = 'Hola soy Brayan'
        
        const { getByText } = render( <PrimeraApp saludo={ saludo } />);

        expect( getByText(saludo)).toBeInTheDocument()

     })
      */

     test('debe de mostrar <PrimeraApp /> correctamente', () => {
         const saludo = 'Hola, soy Brayan'
         const wrapper = shallow(<PrimeraApp saludo = {saludo} /> )

         expect( wrapper ).toMatchSnapshot();
     })

     test('debe de mostrar el subtituo enviado por props', () => {
         const saludo = 'Hola, soy Brayan';
         const subTitulo = 'Soy un subtitulo';

         const wrapper = shallow(
            <PrimeraApp 
            saludo={ saludo }
            subtitulo={ subTitulo }
            />
         )

         const textParrafo = wrapper.find('p').text();
         expect(textParrafo).toBe( subTitulo )
     })
     
     
 })
 