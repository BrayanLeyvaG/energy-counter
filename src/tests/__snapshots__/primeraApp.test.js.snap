// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <Primera App /> debe de mostrar <PrimeraApp /> correctamente 1`] = `
<Fragment>
  <h1>
    Hola, soy Brayan
    !!!
  </h1>
  <p>
    Soy un subtitulo
  </p>
</Fragment>
`;
